### PR TITLE
feat: change terminal time stamp color from bug bright red to improve readibility

### PR DESCRIPTION
**Pull request by ve-varun-sharma**
## Pull Request Type

Dictionary of Definitions here:
https://ben.balter.com/2015/12/08/types-of-pull-requests/

- [] Just a heads up 🙆
- [ ] Sanity check 🏃‍♂️
- [ ] Work in progress (WIP) 🏗️
- [ ] Early feedback ✍️
- [x] Line-by-line review 📝
- [ ] Pull request to a pull request 🔀

## Description

The terminal color change should improve Readability, reduced Confusion via by avoiding the use of red for timestamps, we can minimize the risk of false positives and streamline the debugging process, and overall abide by terminal UX best practices from what I understand about the objective of the time stamp purpose in the defang project.

This PR addresses this Github issue: https://github.com/DefangLabs/defang/issues/707

#### Caveats/Notes (optional)

I was not able to fully test the terminal UI as I did not run this specific defang version. Instructions on how to best run a local emulated version from VS code would be fantastic too! 

### Screenshots (optional)

[Upload optional screenshots]

## Checklist before requesting a review

- [x] I have performed a self-review of my code
- [x] If it is a core feature, I have added thorough tests.
- [x] Extended the README / documentation, if necessary
- [x] My changes generate no new warnings
- [x] Is the code readable? Prefer to see more self-explanatory and consistency
- [ ] Is the code tested properly? i.e. where appropriate, we're testing well-explained failures, behaviours, fixes, etc

### CODE REVIEW

**Comment by lionello** [comment]
I agree there's improvements to be made here. Timestamps are *Red* for error logs, which are the only ones you'll see when you omit `--verbose`.

File: src/pkg/cli/tail.go [hunk 1]
```diff
@@ -331,7 +331,7 @@ func tail(ctx context.Context, client client.Client, params TailOptions) error {
 				tsColor = termenv.ANSIWhite
 			}
 			if e.Stderr {
-				tsColor = termenv.ANSIBrightRed
+				tsColor = termenv.ansiCyan
```
Comment (lionello) [inline src/pkg/cli/tail.go:334]: It's red because the log comes from `stderr`

**Review by lionello — CHANGES_REQUESTED** [review]

**Comment by lionello** [comment]
@ve-varun-sharma closing this because we did make some changes around our verbosity. Also "cyan" was already used for another column. I think there's something to be said to make timestamps optional, then this would not be an issue.

**Comment by ve-varun-sharma** [comment]
> @ve-varun-sharma closing this because we did make some changes around our verbosity. Also "cyan" was already used for another column. I think there's something to be said to make timestamps optional, then this would not be an issue.

That definitely makes sense to me! Thanks for the update here :) 🙏🏽

**Pull request closed without merging**